Sources/SpeziOnboarding/SignatureView.swift

```diff
@@ -42,6 +43,11 @@ public struct SignatureView: View {
                 CanvasView(drawing: $signature, isDrawing: $isSigning, showToolPicker: .constant(false))
                     .accessibilityLabel(Text("SIGNATURE_FIELD", bundle: .module))
                     .accessibilityAddTraits(.allowsDirectInteraction)
+                    .onPreferenceChange(CanvasView.CanvasSizePreferenceKey.self) { size in
+                        // for some reason, the preference won't update on visionOS if placed in a parent view
+                        print("Canvas size inner is \(size)")
```

Comment:
Should we keep this `print` statement in there? I would opt for either removing it or opting for a debug `Logger` log

Reply:
We might just remove that 👍